### PR TITLE
Improved preamble article.

### PR DESCRIPTION
**Pull request by Giggitybyte**
# Summary
Replaced the .NET version section of the preamble with a table for each DSharpPlus branch.

### CODE REVIEW

File: docs/articles/preamble.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ uid: articles.preamble
 title: Article Preamble
 ---
 
-> [!IMPORTANT]
```
Comment (Plerx2493) [inline docs/articles/preamble.md:6]: This is intentionally a big red box and even now many people dont read it

Comment (Giggitybyte) [reply docs/articles/preamble.md:6]: Perhaps the wording of that could be improved too.

File: docs/articles/preamble.md [hunk 2]
```diff
@@ -20,28 +20,25 @@ you're not sure what to do first, Bob's tutorial series should be your starting
...
+| :---------- | :-----: | :-----: | :-----: | 
+| [Stable][17], `v4.5.X` | `v8.0` - `v9.0`</br>✔️ |  `v3.1`</br>⚠️ | `v4.6.1` - `v4.8.1`</br>⚠️ |
+| [Nightly][18], `v5.0` | `v9.0`</br>✔️ | `v3.1`</br>⚠️ | ❌ |
+| [Future][19], `v6.0` | `v9.0`</br>✔️ | ❌ | ❌ |
 
-#### Latest Stable 4.5.0
+<sub> ✔️ `Recommended and supported`  &nbsp;●&nbsp; ⚠️ `Unsupported, might still work` &nbsp;●&nbsp; ❌ `Unsupported, do not use`</sub>
 
-Version 4.5.0 *should* work on .NET and .NET Core versions after 3.1 and is supported from version 8.0 upwards.
+Generally, you should be targeting the latest LTS version of .NET.
```
Comment (akiraveliara) [inline docs/articles/preamble.md:35]: D#+ doesn't target LTS, so this is rather silly

File: docs/articles/preamble.md [hunk 3]
```diff
@@ -20,28 +20,25 @@ you're not sure what to do first, Bob's tutorial series should be your starting
...
-moment. .NET Framework 4.6.1 through 4.8.1 *should* work, but we will not provide support or fixes for any issues arising there.
-Additionally, if you are using a game engine with C# support (such as Unity), you should consider using the [Discord GameSDK][8]
-instead of DSharpPlus.
+See the table below for supported [.NET implementations][16]:
 
-If you use unsupported software, you are on your own with any arising issues.
+| DSharpPlus Branch | .NET | .NET Core | .NET Framework |
+| :---------- | :-----: | :-----: | :-----: | 
+| [Stable][17], `v4.5.X` | `v8.0` - `v9.0`</br>✔️ |  `v3.1`</br>⚠️ | `v4.6.1` - `v4.8.1`</br>⚠️ |
+| [Nightly][18], `v5.0` | `v9.0`</br>✔️ | `v3.1`</br>⚠️ | ❌ |
```
Comment (akiraveliara) [inline docs/articles/preamble.md:30]: nightly will not work on core 3.1

File: docs/articles/preamble.md [hunk 4]
```diff
@@ -20,28 +20,25 @@ you're not sure what to do first, Bob's tutorial series should be your starting
...
+| [Future][19], `v6.0` | `v9.0`</br>✔️ | ❌ | ❌ |
```
Comment (akiraveliara) [inline docs/articles/preamble.md:31]: nit: v6 targets 10.0 alpha builds, not important since v6 isn't functional

**Review by akiraveliara — CHANGES_REQUESTED** [review]
in addition to plerx' comment, there is incorrect information

File: docs/articles/preamble.md (hunk 1 above)
Comment (akiraveliara) [reply docs/articles/preamble.md:6]: i do think red triggers more of a "this is important" connotation, but, no strong opinions

**Review by akiraveliara — APPROVED** [review]

**Review by VelvetToroyashi — APPROVED** [review]

**Review by Naamloos — APPROVED** [review]

**Comment by Naamloos** [comment]
LGTM. I do think the note should be red instead of blue personally. It's more of a warning: "beware! this will not work with the stable releases!!"

**Comment by Naamloos** [comment]
might*

**Pull request merged**